packages/logger/package.json

```diff
@@ -34,7 +34,8 @@
         "@types/debug": "4.1.7",
         "@types/lodash": "4.14.176",
         "@types/vscode": "1.63.1",
-        "jest-extended": "1.2.0",
+        "jest-extended": "2.0.0",
+        "jest": "27.5.1",
```

Comment:
I know we get a complaint about missing peer dependencies here, but can we get away with not having a specific jest version for this package (and store)?

Reply:
`"jest": ">=27.2.5",`
Seems to work better as the version from the root will be used and the warning is gone. 
Updated in 05539429acb5bcbd25dbe45d7fb97a177de7bb51